e2e/config/config.go

```diff
@@ -389,6 +402,40 @@ func resourceName(url string) string {
 	return strings.ToLower(resourceNameForbiddenCharacters.ReplaceAllString(url, "-"))
 }
 
+// validateDeployers checks that the deployers are configured correctly.
+func validateDeployers(config *Config, options Options) error {
```

Comment:
can consider moving "validateDeployers" and "validateDuplicateDeployerNames" closer to validateTests, since they all perform config validation and are logically related.